telethon/telegram_client.py

```diff
@@ -302,9 +304,20 @@ def get_dialogs(self,
             [utils.find_user_or_chat(d.peer, entities, entities) for d in ds]
         )
 
-    # endregion
+    def get_drafts(self):  # TODO: Ability to provide a `filter`
+        """
+        Gets all open draft messages.
+
+        Returns a list of custom `Draft` objects that are easy to work with: You can call
```

Comment:
Maybe we can mention "see the draf class for more" or "type help(telethon.tl.custom.Draft)" (it's really useful and I think little people know of that feature).

Or maybe we should generate a page with the docs like most sites do? It really is useful if docstrings do have examples and more stuff, otherwise it's too technical.